llvm/test/Feature/Repo/repo_pruning_llvm.global_ctors.ll

```diff
@@ -0,0 +1,55 @@
+; This testcase tests the program repository pruning pass for the llvm.global_ctors.
+;
+; The testcase includes three steps:
+; Step 1: Build the code and create the database 'clang.db' which contains all Tickets.
```

Comment:
I'd remove "which contains all tickets". The database contains the result of the compilations; I don't think we need to explain here.

Reply:
Yes, removed.